app/src/main/java/org/mifos/mobile/ui/beneficiary/presentation/Icons.kt

```diff
@@ -0,0 +1,89 @@
+package org.mifos.mobile.ui.beneficiary.presentation
+
+import android.content.res.Configuration
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import org.mifos.mobile.R
+import org.mifos.mobile.core.ui.theme.MifosMobileTheme
+
+@Composable
+fun Icons(
```

Comment:
naming convention, I don't what is Icons meaning here, please give screen section wise name.

Reply:
![Screenshot (117)](https://github.com/openMF/mifos-mobile/assets/90254566/3c7dd9f6-4d67-441c-b1ee-49afd1868531)